### ISSUE TITLE
Convert scripts in `developer/bin` to Homebrew commands

### ISSUE THREAD

**Issue by bevanjkay** [Homebrew/homebrew-cask#174192]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [ ] I have retried my command with `--force`.
- [ ] I ran `brew update-reset && brew update` and retried my command.
- [ ] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

-

### Command that failed

-

### Output of command with `--verbose --debug`

```shell
-
```

### Output of `brew doctor` and `brew config`

```shell
-
```

### Output of `brew tap`

```shell
-
```

**Comment by bevanjkay** [comment]
@Homebrew/cask
Interested in some feedback here to see what is used often.

I am not sure how many of the helpers we need to keep for building casks, as it seems most "ids" and "loaded items" are discovered on CI anyway, which I personally don't have a problem with.

**Comment by p-linnane** [comment]
@Bo98 advised that we can probably use https://github.com/mrkn/pycall.rb for the fonts one.

**Comment by Bo98** [comment]
Is the proposal to move them to `$(brew --repo homebrew/cask)/cmd`? If so: makes sense.

Some of the more generic ones could perhaps be in Homebrew/brew, with modifications. For example, integrating `generate_cask_token` into `brew create` (and/or an audit) probably makes sense.

**Comment by bevanjkay** [comment]
> Is the proposal to move them to `$(brew --repo homebrew/cask)/cmd` (like homebrew-core's rebottle command)? If so: makes sense.

Yeah, this exactly.

**Comment by tshu-w** [comment]
For those who wants to use these commands, it should tap cask first.
```console
brew tap --force homebrew/cask
```

**Comment by tae8838** [comment]
Can I help remove the commands on Potentially Remove list and migrate `list_loaded_kext_ids`?

**Comment by bevanjkay** [comment]
Sounds good to me. For any commands that are removed/updated, we need to ensure that any references in the Cask Cookbook are also removed/updated. https://docs.brew.sh/Cask-Cookbook

**Comment by injust** [comment]
https://docs.brew.sh/Cask-Cookbook#uninstall-quit mentions the `list_running_app_ids` and `list_ids_in_app` developer scripts, but they no longer exist in https://github.com/Homebrew/homebrew-cask/tree/main/developer/bin and the links are dead.

It looks like the former was migrated to a command in https://github.com/Homebrew/homebrew-cask/commit/34f59af1ea927d8590eb29a216d69ac49e058ca8.

**Comment by Bo98** [comment]
Docs are indeed outdated. You can do `brew list-running-app-ids` and `brew list-ids-in-app`

**Comment by injust** [comment]
Someone helped me figure out the commands in another issue. I believe the underscores are hyphens now. You also need to tap homebrew-cask first, and might need `brew developer on` (not sure).

**Comment by Bo98** [comment]
Yes, sorry fixed it to hyphens in an edit.

The tapping homebrew-cask first is not new - it was the same before with `developer/bin`.
